components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/typing.scala

```diff
@@ -167,15 +167,27 @@ object typing {
     override val display = "Null"
   }
 
-  // Unknown is representation of TypedUnion of all possible types
-  case object Unknown extends TypingResult {
-    override def withoutValue: Unknown.type = Unknown
+  sealed trait DisplayStrategy {
+    val display: String
+  }
 
-    override val valueOpt: None.type = None
+  case object UnknownDisplayStrategy extends DisplayStrategy {
```

Comment:
Maybe let's call it `DefaultDisplayStrategy`. Let it be easy to change the implementation of this strategy.